packages/cap-config-writer/test/unit/cap-config/index.test.ts

```diff
@@ -17,7 +17,7 @@ describe('Test enableCdsUi5Plugin()', () => {
             },
             'workspaces': ['app/*'],
             'devDependencies': {
-                'cds-plugin-ui5': '^0.6.13'
+                'cds-plugin-ui5': '^0.9.3'
```

Comment:
```suggestion
                'cds-plugin-ui5': '^0.1.0'
```
Don't this should be updated. The point of this test was that it would be updated by the writer.